enhancements/rhcos/split-rhcos-into-layers.md

```diff
@@ -398,6 +398,33 @@ As such, the versioning change should be transparent to end-users, but on the
 off chance that a customer is relying on the current semantics, we should still
 announce it in the GA release notes.
 
+### /etc/os-release
+
+Previously, the node image used a heavily customized `/etc/os-release`, mutating
+some key fields inherited from RHEL. A major theme of this enhancement and more
+largely of the Image Mode effort is re-emphasizing the fact that RHCOS simply
+builds on top of RHEL Image Mode. As such, many fields in `/etc/os-release` are
+now left untouched.
+
+Important fields that **will be different**:
+
+Key|Old Value|New Value
+---|---|---
+VERSION|419.94.202412101440-0|9.6.20250203-0 (Plow)
```

Comment:
Only thing I'm a little wary of is the text string here `(Plow)`. I know that matches what RHEL does but could cause some issues for anyone relying on it only having the version itself in there.

That being said, if we think this is the right thing to do then now is the time to make that change.

Reply:
I hope that nothing is parsing this today. They should instead be parsing `VERSION_ID` and/or `OSTREE_VERSION` which are meant to be machine-parsable.